pkg/storaged/resize.jsx

```diff
@@ -335,22 +397,22 @@
         passphrase_fields = existing_passphrase_fields(_("Resizing an encrypted filesystem requires unlocking the disk. Please provide a current disk passphrase."));
 
     if (usage.length == 0 && size_fields.length + passphrase_fields.length === 0) {
-        return lvol_and_fsys_resize(client, lvol, shrink_size, false, null);
+        return lvol_or_part_and_fsys_resize(client, lvol_or_part, shrink_size, false, null);
     }
 
     const dlg = dialog_open({
-        Title: _("Shrink logical volume"),
+        Title: title,
         Teardown: TeardownMessage(usage),
         Fields: size_fields.concat(passphrase_fields),
         Action: {
             Title: _("Shrink"),
             action: function (vals) {
                 return teardown_active_usage(client, usage)
                         .then(function () {
-                            return (lvol_and_fsys_resize(client, lvol,
-                                                         to_fit ? shrink_size : vals.size,
-                                                         to_fit ? false : info.shrink_needs_unmount,
-                                                         vals.passphrase || recovered_passphrase)
+                            return (lvol_or_part_and_fsys_resize(client, lvol_or_part,
+                                                                 to_fit ? shrink_size : vals.size,
+                                                                 to_fit ? false : info.shrink_needs_unmount,
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19385-20230928-092115-8ec73ebe-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)